controllers/userController.js

```diff
@@ -1,6 +1,8 @@
 const User = require("../models/users");
 const sequelize = require("../database/db");
 const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const jwtSecret = "akash@456$inno";
```

Comment:
move the secret to the env file, for this, you can use https://www.npmjs.com/package/dotenv